app/models/vacation.rb

```diff
@@ -1,9 +1,12 @@
 class Vacation < ApplicationRecord
   enum status: { pending: 0, approved: 1, rejected: 2 }
+  enum type: { 'planned vacation': 0, 'sick leave': 1, 'unpaid day offs': 2 }
   belongs_to :user
   validates_presence_of :start_date, :end_date
   validate :date_is_valid?
 
+  self.inheritance_column = :_type_disabled
```

Comment:
this is really minor, but most developers prefer to put such things in the beginning of the class for better code readability. The same as including other modules. This line changes the standard behavior of rails model.